app/models/user_provided_service.rb

```diff
@@ -0,0 +1,22 @@
+module UserProvidedService
```

Comment:
since this is a module, what do you think about making it a concern? (concerns are always modules)  http://wendybeth.ghost.io/adding-modules-to-rails-4-models/